024-DoYouEvenSynapse/README.md

```diff
@@ -31,4 +31,5 @@ The purpose of this What The Hack is to learn:
 ## Contributors
 - [James Jones](https://github.com/JamJarchitect/)
 - [Art Rask](https://github.com/artraskmsft)
-- [Orrin Edenfield](https://github.com/OrrinEdenfield)
\ No newline at end of file
+- [Orrin Edenfield](https://github.com/OrrinEdenfield)
+- [Frank Lopez]
```

Comment:
Can't have Frank's name wrap in square brackets if it's not going to be linked.

Just remove the brackets and it should render properly.